jina/helper.py

```diff
@@ -128,6 +129,48 @@ def wrapper(*args, **kwargs):
     return deco
 
 
+def deprecated_class(dep_class=None, new_class=None):
+    """
+    Usage, new_class is a new class.
```

Comment:
It took me some playing around to get what this decorator does. Really nice thing. Anyhow, I'd like to improve the docstring, since it is not obvious, that during runtime the old class will not even be there and replaced completely by the new one. Perhaps something like
```
"""
After applying `deprecated_class` the `dep_class` will only be an alias for the `new_class`.
Any code inside of `dep_class` will be completely overwritten.
"""
```